src/term_rst_post/ansimotd.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright 2021-2021 Vrije Universiteit Brussel
+#
+# This file is part of term_rst_post,
+# originally created by the HPC team of Vrije Universiteit Brussel (https://hpc.vub.be),
+# with support of Vrije Universiteit Brussel (https://www.vub.be),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/sisc-hpc/term_rst_post
+#
+# term_rst_post is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v3.
+#
+# term_rst_post is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this code.  If not, see <http://www.gnu.org/licenses/>.
+#
+##
+"""
+Formatting and management of MOTD text files with ANSI escape codes.
+
+@author: Alex Domingo (Vrije Universiteit Brussel)
+"""
+
+import logging
+import os
+import re
+
+from textwrap import indent, wrap
+
+from term_rst_post.exit import error_exit
+
+logger = logging.getLogger()
+
+
+def accomodate_motd(motd_dir, body_file, head_file=None, foot_file=None, foot_link=None, wrap_width=70):
```

Comment:
`motd_dir` is unused? Or I don't see it ?

Reply:
Indeed, it was a leftover from https://github.com/sisc-hpc/term-rst-post/pull/1/commits/5f67a0313acd4378f7aa249eb640d2d29c5ad6f0. Now it is totally removed in https://github.com/sisc-hpc/term-rst-post/pull/1/commits/afc8ec7bb7c1404633c7e7b16e9dcbad68f37944